index.html

```diff
@@ -1991,7 +2028,8 @@ <h1>Core Representations</h1>
     <p>
 Producers MUST indicate which representation of a document has been used via a media type
 in the document's metadata. Consumers MUST determine which representation a document is in
-via a media type in the document's metadata. Consumers MUST NOT determine the representation
+via the <code>content-type</code> DID resolver metadata field. (See <a href="#did-resolution"></a>).
+Consumers MUST NOT determine the representation
```

Comment:
Any particular reason to use MUST NOT here rather than "Consumers MUST determine the representation using the content-type and content."?

Reply:
This was from a previous pull request #197 that I wrote so I can address it -- what we were trying to do is say that you can't just throw the parser at something to see if it fails or not. So for example you can't get something, throw it in your JSON parser, and then look for the context field to figure out if it's JSON LD. By this definition, it's not JSON LD unless it's got the appropriate JSON LD content type associated with it.

Reply:
Makes sense, thanks for the clarification. I was a bit confused about how to test this, but now that you point out that case it makes sense that the test would need to check that the JSON-LD aspects are valid.